src/models/files/ZosPackageFile.js

```diff
@@ -0,0 +1,113 @@
+import _ from 'lodash'
+import ZosNetworkFile from './ZosNetworkFile'
+import { Logger, FileSystem as fs } from 'zos-lib'
+
+const log = new Logger('ZosPackageFile')
+
+export default class ZosPackageFile {
+
+  constructor(fileName = 'zos.json') {
+    this.fileName = fileName
+    this.data = fs.parseJsonIfExists(this.fileName) || {};
+  }
+
+  exists() {
+    return fs.exists(this.fileName)
+  }
+
+  ifMissingThrow(msg) {
+    if(!this.exists()) throw Error(msg)
+  }
+
+  get lib() {
+    return this.data.lib
+  }
+
+  get name() {
+    return this.data.name
+  }
+
+  get version() {
+    return this.data.version
+  }
+
+  get stdlib() {
+    return this.data.stdlib
+  }
+
+  get stdlibName() {
+    return this.stdlib.name
+  }
+
+  get stdlibVersion() {
+    return this.stdlib.version
+  }
+
+  get contracts() {
+    return this.data.contracts
+  }
+
+  get contractAliases() {
+    return Object.keys(this.contracts)
+  }
+
+  contract(alias) {
+    return this.contracts[alias]
+  }
+
+  isLib() {
+    return !!this.lib
+  }
+
+  hasName(name) {
```

Comment:
Same comment as in `ifThrow...`. Why not just let the caller make the comparison, so we don't need yet another method here?

Reply:
The idea of encapsulating this behavior in a method is not to repeat it every time. If we don't have this method we will be repeating the comparison `.name === x` in many places. Of course, we could work with JS plain objects, but I think if we do this way we will reduce a lot the amount of duplicated code and it will let us refactor/modify that code in a faster way